src/components/Loader/Loader.tsx

```diff
@@ -32,6 +16,88 @@ type Props = {
 }
 type ElementProps = Omit<HTMLAttributes<HTMLDivElement>, keyof Props>
 
+const loaderStyle = tv({
+  slots: {
+    wrapper: ['smarthr-ui-Loader', 'shr-inline-block', 'shr-overflow-hidden'],
+    spinner: ['shr-relative', 'shr-block', 'shr-animate-loader-spinner-spin', 'shr-mx-auto'],
+    line: ['shr-absolute', 'shr-block', 'shr-w-full', 'shr-h-full', 'shr-opacity-0'],
+    cog: [
+      'shr-inline-block',
+      'shr-relative',
+      'shr-w-1/2',
+      'shr-h-full',
+      'shr-overflow-hidden',
+      'shr-border-inherit',
+    ],
+    cogInner: [
+      'shr-absolute',
+      'shr-top-0',
+      'shr-left-0',
+      'shr-block',
+      'shr-w-[200%]',
+      'shr-h-full',
+      'shr-box-border',
+      'shr-border-solid',
+      'shr-border-inherit',
+      'shr-border-b-transparent',
+      'shr-rounded-[50%]',
+    ],
+    textSlot: ['shr-block', 'shr-mt-1', 'shr-text-base', 'shr-text-center'],
```

Comment:
元にはなさそうに見えた（VRT で diff が出てるのもたぶんここ）

```suggestion
    textSlot: ['shr-mt-1', 'shr-text-base', 'shr-text-center'],
```

Reply:
あ、申し送り忘れてましたが、ここは元の実装が inline 要素に対して margin-top が指定されていた関係で margin が効いてなかったので修正した感じです！
なので、VRT の diff は意図したものでした！